test/pkg/firewall/firewall.go

```diff
@@ -0,0 +1,212 @@
+package firewall
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/openshift-kni/commatrix/pkg/client"
+	"github.com/openshift-kni/commatrix/pkg/consts"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+
+	"github.com/openshift-kni/commatrix/pkg/types"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+func ApplyRulesToNode(cs *client.ClientSet, ns string, NFTtable []byte, nodeName string, wg *sync.WaitGroup) error {
+	defer wg.Done()
+
+	utilsHelpers := utils.New(cs)
```

Comment:
this also needs to be in the firewall struct so you can have mocks for unit-tests in the future